cmd/gitserver/main.go

```diff
@@ -76,7 +80,13 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to initialize database stores: %v", err)
 	}
+
 	repoStore := database.Repos(db)
+	codeintelDb := codeinteldbstore.NewWithDB(db, &observation.Context{
```

Comment:
```suggestion
	codeintelDB := codeinteldbstore.NewWithDB(db, &observation.Context{
```

For naming consistency 😁